router.go

```diff
@@ -395,6 +402,72 @@ func (app *App) register(methods []string, pathRaw string, group *Group, handler
 	}
 }
 
+func (app *App) routeExists(method, pathRaw string) bool {
```

Comment:
Something is wrong with this method. It's causing over 200 unit-tests to fail now.